src/main.h

```diff
@@ -135,6 +135,8 @@ static const bool DEFAULT_ADDRESSINDEX = false;
 static const bool DEFAULT_TIMESTAMPINDEX = false;
 static const bool DEFAULT_SPENTINDEX = false;
 
+static constexpr unsigned int DEFAULT_MAX_MEMPOOL_SIZE_MB = 400;
```

Comment:
What about moving this constant to file `txmempool.h`? It seems to me to be more scoped in that way.
I've checked, there should be no building conflict doing this (in case you agree, please double check if you can get rid of now-became-unused includes introduced by this PR, like for example for file `src/test/policyestimator_tests.cpp`).

Reply:
I agree with you it would be a great idea. But, unfortunately, right now we have all similar variables (such as `DEFAULT_MAX_ORPHAN_TRANSACTIONS`) defined in `main.h`.
I believe we really should refactor things at some point, and move these variables where they actually belong, in an organized way. But for the time being I'd rather be consistent, and leave it here.

Reply:
Agreed to handle this in a follow-up PR.